doc/regexp/unicode_properties.rdoc

```diff
@@ -0,0 +1,590 @@
+== \Regexps Based on Character Properties
+
+- <tt>/\p{Alpha}}/</tt>
+- <tt>/\p{Blank}/</tt>
+- <tt>/\p{Cntrl}/</tt>
+- <tt>/\p{Digit}/</tt>
+- <tt>/\p{Graph}/</tt>
+- <tt>/\p{Lower}/</tt>
+- <tt>/\p{Print}/</tt>
+- <tt>/\p{XPosixPunct}/</tt>
+- <tt>/\p{Space}/</tt>
+- <tt>/\p{Upper}/</tt>
+- <tt>/\p{XDigit}/</tt>
+- <tt>/\p{Word}/</tt>
+- <tt>/\p{Alnum}/</tt>
+- <tt>/\p{ASCII}/</tt>
+- <tt>/\p{Punct}/</tt>
+- <tt>/\p{Any}/</tt>
+- <tt>/\p{Assigned}/</tt>
+- <tt>/\p{C}/</tt>
+- <tt>/\p{Cf}/</tt>
+- <tt>/\p{Cn}/</tt>
+- <tt>/\p{Co}/</tt>
+- <tt>/\p{Cs}/</tt>
+- <tt>/\p{L}/</tt>
+- <tt>/\p{LC}/</tt>
+- <tt>/\p{Ll}/</tt>
```

Comment:
maybe these would be more intelligible if they used the long form, e.g. `\p{Lowercase_Letter}`?

Reply:
Added.

Reply:
Both forms now in revised list.